### DISCUSSION TITLE
Logamatic 4211

### DISCUSSION THREAD

**Discussion by silfax** [emsesp/EMS-ESP32#1022] [General]
Hi,
Would it be possible to have it working with an older Logamatic 4211? Communication is based on the ECO-CAN bus, which seems to be a different thing(?) from the EMS-BUS. Thanks.

**Comment by proddy** [comment]
EMS-ESP only works with the EMS bus I'm afraid.

**Comment by bbqkees** [comment]
This Logamatic controller is not supported because it does not have an EMS bus.
